examples/keyvalue-inmemory/main.go

```diff
@@ -0,0 +1,74 @@
+//go:generate wit-bindgen-wrpc go --out-dir bindings --package github.com/wasmCloud/provider-sdk-go/examples/keyvalue-inmemory/bindings wit
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/wasmCloud/provider-sdk-go"
+	server "github.com/wasmCloud/provider-sdk-go/examples/keyvalue-inmemory/bindings"
+	"golang.org/x/sync/errgroup"
+)
+
+func main() {
+	providerGroup := new(errgroup.Group)
+	p, err := provider.New(
```

Comment:
This is missing unsubscribe logic to unsubscribe from the topics on exit, right?